### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Wishlist fuctionality completed.

### CODE REVIEW

File: frontend/src/components/card/CartCard.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { useCart } from "context/cartContex";
+import { useWishlist } from "context/wishlistContex";
 import React from "react";
```
Comment (vishalg8454) [inline frontend/src/components/card/CartCard.js]: typo in Context

File: frontend/src/pages/wishlist/Wishlist.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            {wishlist.map((item) => (
+              <WishlistCard
+                key={item._id}
+                id={item._id}
+                img={item.image}
+                name={item.name}
+                owner={item.owner}
+                ratings={item.numOfReviews}
+                price={item.price}
+              />
```
Comment (vishalg8454) [inline frontend/src/pages/wishlist/Wishlist.js:27]: destructuring can be used

**Review by vishalg8454 — APPROVED** [review]
Code is easy to understand, good use of Contexts. Just a minor feedback, can merge.

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me. Just make the changes suggested by Visha;

**Pull request merged**